### PR TITLE
colorcet 3.0.1

### PR DESCRIPTION
**Pull request by skupr-anaconda**
**Jira ticket:** [PKG-672](https://anaconda.atlassian.net/browse/PKG-672) (colorcet-3.0.1)

**The upstream data:**
Github releases:  https://github.com/holoviz/colorcet/releases
[Diff between the latest and previous upstream releases](https://github.com/holoviz/colorcet/compare/3.0.0...3.0.1)
Requirements:
 * setup.py:  https://github.com/holoviz/colorcet/blob/v3.0.1/setup.py
 *  pyproject.toml:  https://github.com/holoviz/colorcet/blob/v3.0.1/pyproject.toml

**_Actions:_**

1. Add `entry_points`
2. Remove `param` from `host` and `run`
3. Add the `colorcet` command

**Package's statistics**
<details>

 * Priority B | effort: easy | Category: anaconda_affiliated | subcategory:  | pkg_type: python
 * Outdated platfroms: 'linux-64', 'linux-aarch64', 'linux-ppc64le', 'osx-64', 'osx-arm64', 'win-64'
 * Latest version: 3.0.1
 * Release on PyPi:
    * version: 3.0.1
    * date: 2022-10-03T15:30:05
* [PyPi history](https://pypi.org/project/colorcet/#history)
 * Popularity: 
    * 3 months downloads: 266810.0
    * All time:  695035 downloads -  colorcet  3.0.1  Collection of perceptually uniform colormaps  

</details>

**Links:**
<details>

* [The v3.0.1 upstream URL](https://github.com/holoviz/colorcet/tree/v3.0.1)
* [Bug tracker](https://github.com/holoviz/colorcet/issues)
* [PyPi](https://pypi.org/project/colorcet)
* [conda-forge recipe](https://github.com/conda-forge/colorcet-feedstock/blob/master/recipe/meta.yaml)
* [Anaconda recipe feedstock](https://github.com/AnacondaRecipes/colorcet-feedstock)
* [Update branch: _3.0.1_](https://github.com/AnacondaRecipes/colorcet-feedstock/tree/3.0.1)
* [Diff between upstream conda-forge **main** and aggregate **feature** branch](https://github.com/conda-forge/colorcet-feedstock/compare/main...AnacondaRecipes:3.0.1#files_bucket)
* [Diff between origin aggregate **master** and aggregate **feature** branch](https://github.com/AnacondaRecipes/colorcet-feedstock/compare/master...AnacondaRecipes:3.0.1#files_bucket)
* [The last merged PRs](https://github.com/AnacondaRecipes/colorcet-feedstock/pulls?q=is%3Apr+is%3Amerged+sort%3Aupdated-desc+)

</details>

**Other checks:**

<details>

1. - [x] https://github.com/holoviz/colorcet/tree/v3.0.1 exists
4. - [x] https://colorcet.holoviz.org/index.html is present
5. - [x] Check the pinnings
7. - [x] Additional research
    https://github.com/holoviz/colorcet/issues
    200
8. - [x] `dev_url` is present
    https://github.com/holoviz/colorcet
9. - [x] `doc_url` is present
    https://colorcet.holoviz.org/user_guide/index.html
7. - [x] `doc_source_url` is present
    https://github.com/holoviz/colorcet/tree/master/doc
10. - [x] Verify that the `build_number` is correct
11. - [x] has `setuptools`
12. - [x] has `wheel`
13. - [x] `pip` in test
14. - [x] Verify the test section
15. - [x] Verify if the package is `architecture specific`
16. - [x] Verify that private modules are not mentioned in the recipe For example: (_private_module)
17.  - [x] license_file: LICENSE.txt is present
18. - [x] License: CC-BY-4.0
    - [ ] License is `spdx` compliant
 * Check if the license identifier has correct name from the SPDX License List

| Identifier   |
|:-------------|
| CC-BY-4.0    |

19. - [x] license_family CC is present
</details>

**Updating the recipe:**

If the recipe needs additional modification the update branch can be modified.

```
git clone -b 3.0.1 git@github.com:AnacondaRecipes/colorcet-feedstock.git
```

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,28 +12,29 @@ source:
...
 test:
-  requires:
-    - pip
   imports:
     - colorcet
+  requires:
+    - pip
   commands:
     - pip check
+    - colorcet --help
```
Comment (katietz) [inline recipe/meta.yaml:37]: this recipe has this superflous doc_source_url already ... we should remove it.  there is no value in having it togehter with doc_url.  btw there is also this insane license_url in combination with license_file.  Please remove license_url here too.

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:37]: I addressed your comment

**Review by katietz — APPROVED** [review]
Thanks!

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**